packages/pn-commons/src/components/NotificationDetail/__test__/NotificationPaymentRecipient.test.tsx

```diff
@@ -17,12 +17,15 @@ describe('NotificationPaymentRecipient Component', () => {
     f24Only: getF24Payments(payments, 0),
   };
 
+  const F24TIMER = 15000;
+
   it('should render component title and subtitle', () => {
```

Comment:
In one of these test add the check that it is shown the correct f24 view (f24 only and f24 attachment)

Reply:
There's already a test on NotificationPaymentF24Item.test.tsx

Reply:
yes. here check only that, based on payment structure, it is shown the correct view